### PR TITLE
just a few changes surrounding pipelines, formatters and sinks

### PR DESCRIPTION
**Pull request by jimlambrt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,35 +11,70 @@ While this library is fully open source and HashiCorp will be maintaining it (si
...
+Clients interact with the library via the Broker by sending events. A Broker
+processes incoming Events, by sending them to the pipelines (graphs) associated
+with the Event's type.  A given Broker, along with its associated set of
+pipelines (graphs), will be configured programmatically. 
 
 
 ## Nodes 
 
-A Node is a node in a Pipeline, that can perform any kind of operation that it wants to on an Event.  A node has a Type, one of: Processor, Formatter, Sink.
+A Node is a node in a Pipeline, that can perform operations on an Event.  A node
```
Comment (s-christoff) [inline README.md:36]: Rereading this I realize it doesn't really help a lot.
But I don't know how to rephrase it.
Maybe we should copy that diagram from the RFC?

**Review by s-christoff — DISMISSED** [review]
Only one minor nit but it's my own words so 🤷‍♀️

File: README.md (hunk 1 above)
Comment (jimlambrt) [reply README.md:36]: Hum... the pic would make things a bit clearer.

Comment (jimlambrt) [reply README.md:36]: I'm adding a pic.

**Pull request merged**